src/wtc-gallery-component.scss

```diff
@@ -62,6 +62,7 @@
   &.is-active {
     opacity: 1;
     transform: scale(1) translateY(-50%);
+    z-index: 1;
```

Comment:
Can you please just add a quick comment here to note why this is here?

Reply:
Done!